CIP-0060/README.md

```diff
@@ -0,0 +1,333 @@
+---
+CIP: 60
+Title: Music Token Metadata
+Authors: Andrew Westberg <awestberg@projectnewm.io>, Ryan Jones <rjones@projectnewm.io>, Justin Morgan <jusemorgan@gmail.com>, Ian Singer <tcl@fre5hmusic.com>, Anthony Eizmendiz <aeizmendiz@icloud.com>, Session Cruz <session@demu.pro>, Jimmy Londo <SickCityCleveland@gmail.com>, Gudbrand Tokerud <Gudbrand.tokerud@gmail.com>, Kevin St.Clair <kos1777@gmail.com>
+Comments-URI: no comments yet
+Status: Draft
+Type: Informational
```

Comment:
Small-note: I believe this is more than informational and fall in the category of _processes_.